src/main/common/router/routerFinder.ts

```diff
@@ -1,8 +1,11 @@
+import * as path from 'path'
 import { Router } from 'express'
 import * as requireDirectory from 'require-directory'
 
+const fileExtension: string = path.extname(__filename).slice(1)
+
 const options: object = {
-  extensions: ['ts'],
+  extensions: [fileExtension],
```

Comment:
could just change this to accept js again, I only removed it last week

Reply:
Did that as a first thing but faced some issue. Because JS files in the same place are TS files (right next to TS files) it `routeFinder` loaded both and then there was compilation error because `ts-node` was unable to work with compiled JS files.